qt/applications/workbench/workbench/utils/windowfinder.py

```diff
@@ -25,6 +25,16 @@ def get_main_window_widget():
             return widget
 
 
+def find_window(object_name, cls=None):
+    """
+        Finds the currently open window
+        :return: window or None
+        """
+    windows = QApplication.topLevelWidgets()
+    for window in windows:
+        if window.objectName() == object_name and isinstance(window, cls):
```

Comment:
I think `isinstance` will throw an error if you pass it `None` as its second argument. Perhaps make the `cls` argument required? Or probably better is to skip the `isinstance` check if no `cls` is given

Reply:
:+1: